flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/TimeFunctionsITCase.java

```diff
@@ -580,10 +585,27 @@ private Stream<TestSetSpec> ceilTestCases() {
                                 $("f2").ceil(TimeIntervalUnit.MILLENNIUM),
                                 "CEIL(f2 TO MILLENNIUM)",
                                 LocalDateTime.of(3001, 1, 1, 0, 0),
-                                TIMESTAMP().nullable()));
+                                TIMESTAMP().nullable())
+                        .testResult(
+                                $("f2").cast(TIMESTAMP_LTZ(3)).ceil(TimeIntervalUnit.MINUTE),
+                                "CEIL(CAST(f2 AS TIMESTAMP_LTZ(3)) TO MINUTE)",
+                                ZonedDateTime.of(
+                                                LocalDateTime.of(2020, 2, 29, 1, 57, 0),
+                                                ZoneId.of("UTC"))
+                                        .toInstant(),
+                                TIMESTAMP_LTZ(3))
+                        .testResult(
+                                $("f3").cast(TIMESTAMP_LTZ(3)).ceil(TimeIntervalUnit.SECOND),
+                                "CEIL(CAST(f3 AS TIMESTAMP_LTZ(3)) TO SECOND)",
+                                ZonedDateTime.of(
+                                                LocalDateTime.of(2020, 2, 29, 1, 56, 58),
+                                                ZoneId.of("UTC"))
+                                        .toInstant(),
+                                TIMESTAMP_LTZ(3)));
     }
 
     private Stream<TestSetSpec> floorTestCases() {
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
```

Comment:
that's weird that we set it for all test cases...
can we set it only for some certain test cases and then unset it back?